### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by bastjan**
## Summary

* Implements an interactive, guided way to download and decrypt emergency credentials.
* Implements a non-interactive/scriptable way to download and decrypt emergency credentials.
* Both the interactive and non-interactive are e2e tested with test credentials and an `expect` script in the `e2e` folder.
* Discovers the bucket configuration from passbolt
* Easy installation using `install.sh`
* Shows tokens on stdout and creates a kubeconfig from them.

## Checklist

- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog
- [x] Update tests.

### CODE REVIEW

File: pkg/config/dir.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	"os"
+	"path/filepath"
+	"runtime"
+)
+
+const (
+	EnvConfigDir = "EMR_CONFIG_DIR"
+
+	appData       = "AppData"
+	localAppData  = "LocalAppData"
```
Comment (simu) [inline pkg/config/dir.go:13]: Unused?

**Review by simu — APPROVED** [review]
LGTM overall. Some nitpicks inline

File: e2e/interactive.tcl [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+# Escape key
+send -- "\x1b"
+send -- ":x\r"
+expect "survey*written"
+log "private_key entry done"
+log_user 1
+
+log "Expecting passphrase prompt"
+expect_prompt "Passbolt passphrase"
+send -- "$passphrase"
```
[the remark this answers is not in the document]
Comment (bastjan) [reply e2e/interactive.tcl:45]: The input library hides the password with `*` as the replacement character while entering.

[the remark this answers is not in the document]
Comment (simu) [reply e2e/interactive.tcl:45]: Ah, that's fine then :)

File: main.go [hunk 3]
```diff
@@ -1,7 +1,396 @@
 package main
 
-import "fmt"
+// A simple program demonstrating the textarea component from the Bubbles
+// component library.
```
[the remark this answers is not in the document]
Comment (bastjan) [reply main.go]: yes 🤦

File: pkg/config/dir.go (hunk 1 above)
Comment (bastjan) [reply pkg/config/dir.go:13]: That would be the data/cache dir for windows. We don't save any state or data so it is unused.

**Pull request merged**